jms/1.7/modules/ROOT/pages/jms-activemq-configuration.adoc

```diff
@@ -1,153 +1,200 @@
 
-If the message has persistent delivery set, ActiveMQ sends the message to `DLQ.QUEUE` (dead letter queue).
+ActiveMQ enables you also to have a client-side redelivery configuration:
 
-=== Advanced Redelivery
+* The *Initial redelivery delay* field enables you to configure in milliseconds how much time to wait for the message to be redelivered the first time. +
+* The *Redelivery delay* field enables you to configure in milliseconds how much time to wait until the message is subsequently redelivered after the first redelivery.
 
-ActiveMQ enables users to have a client-side redelivery configuration. You can also configure the <<poison-messages,maximum times that a message is redelivered>> and how fast the connector is required to redeliver the message.
+To configure these fields in Studio, follow these steps:
 
-`initialRedeliveryDelay`: In milliseconds, how much time to wait for the message to be redelivered the first time.
-`redeliveryDelay`: In milliseconds, how much time to wait until the message is subsequently redelivered after the first redelivery.
+. In Studio, open the *JMS Config* global element window.
+. In the *Connection Factory* section, set the *Factory configuration* field to *Edit inline*.
+. Set the *Initial redelivery delay* field to `1000`.
+. Set the *Redelivery delay* field to `100`.
+. Set the *Max redelivery* field to `10`.
+. Click *OK*.
+
+.JMS Connector Redelivery configuration
+image::jms-activemq-redelivery.png[Set the Max redelivery field to 10]
+
+In the XML editor, the `maxRedelivery`,`redeliveryDelay` and `initialRedeliveryDelay` configurations look like this:
 
 [source,xml,linenums]
 ----
 <jms:config name="JMS_Config">
   <jms:active-mq-connection>
-    <jms:factory-configuration
+    <jms:factory-configuration maxRedelivery="10"
       redeliveryDelay="100"
       initialRedeliveryDelay="1000"/>
   </jms:active-mq-connection>
 </jms:config>
 ----
 
-== Object Serialization
+== Configure Trusted Packages
+
+ActiveMQ versions 5.12.2, 5.13.0, and later restricts the classes that can be serialized and deserialized to prevent the execution of a malicious payload that can exploit the host system.
+
+For ActiveMQ, JMS Connector enables only the JDK and JRE provided classes by default, so if you
+need to exchange object messages, you must add the packages your applications use by activating the *Trusted packages* field. +
+Even though you can also enable the *Trust all packages* field to allow the serialization of more classes by allowing any object to be serialized and deserialized, this is not as secure as leaving it disabled in most cases. Keep this parameter disabled to improve the security and help prevent malicious attacks.
```

Comment:
```suggestion
Even though it is available to you, avoid enabling the *Trust all packages* field, which limits your ability to improve security and prevent malicious attacks.
```
I think that the previous text has explained about the danger of uncontrolled serialization/deserialization, so we can shorten this quite a bit.